cmd/root.go

```diff
@@ -0,0 +1,93 @@
+//
+// Use and distribution licensed under the Apache license version 2.
+//
+// See the COPYING file in the root project directory for full text.
+//
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/jaypipes/ghw"
+	"github.com/spf13/cobra"
+)
+
+var (
+	version   string
+	buildHash string
+	buildDate string
+	debug     bool
+	info      *ghw.HostInfo
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "ghwc",
+	Short: "ghwc - Discover hardware information.",
+	Long: `
+          __
+ .-----. |  |--. .--.--.--.
+ |  _  | |     | |  |  |  |
+ |___  | |__|__| |________|
+ |_____|
```

Comment:
lol, this makes me think of someone giving someone else the middle finger.

Reply:
lolol